exercise_02.py

```diff
@@ -0,0 +1,25 @@
+def reverse(number):
+    n_list = list(str(number))
+    number = ""
+    for i in range(1, len(n_list) + 1):
+        number += n_list[-i]
+
+    return number
```

Comment:
This function should return a number, but you are returning a string... This is very easy to solve.